### PR TITLE
Post-commit review from mike

### PR DESCRIPTION
**Pull request by frankfarzan**

### CODE REVIEW

**Comment by frankfarzan** [comment]
ptal

File: online/shipping-app-backend/quota.yaml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-kind: ResourceQuota
-apiVersion: v1
-metadata:
-  name: pod-quota
```
Comment (mdruskin) [inline online/shipping-app-backend/quota.yaml]: Do we want this to be allowed? Since the quota will be renamed.

File: README.md [hunk 2]
```diff
@@ -1 +0,0 @@
-# foo-corp-example
```
Comment (mdruskin) [inline README.md]: Move the whole thing under a foo-corp subfolder

File: online/shipping-app-backend/quota.yaml (hunk 1 above)
Comment (frankfarzan) [reply online/shipping-app-backend/quota.yaml]: cannot have a resource without name

File: README.md (hunk 2 above)
Comment (frankfarzan) [reply README.md]: done

File: CODEOWNERS [hunk 3]
```diff
@@ -1,3 +0,0 @@
-/ @frankfarzan
```
Comment (briantkennedy) [inline CODEOWNERS]: Add a CODEOWNERS or OWNERS file example at a lower tree node

**Pull request closed without merging**